src/library/F3DLoader.cxx

```diff
@@ -393,9 +402,16 @@ bool F3DLoader::LoadFile(int load)
     this->Internals->Options = this->Internals->Parser.GetOptionsFromConfigFile(filePath);
   }
 
-  F3DLog::SetQuiet(this->Internals->Options.Quiet);
+  // Start working with libf3d API here
+  // This should change in the future and be much higher up
+  // TODO
+  F3DOptionsParser::ConvertToNewAPI(this->Internals->Options, &this->Internals->NewOptions);
```

Comment:
So is it acceptable like this for now ?

Doing better would require to split F3DLoader in two, which is planned but will be a bigger endeavour.

@Meakk

Reply:
It's fine if we have plan to improve it.